src/components/Select/Select.tsx

```diff
@@ -166,6 +166,7 @@ export function Select(props: SelectProps) {
     className,
     children,
     'aria-label': ariaLabel,
+    value = null,
```

Comment:
With HeadlessUI update, this complains if it's empty that it's changing from uncontrolled to controlled